example/src/components/router-head/router-head.tsx

```diff
@@ -18,7 +18,10 @@ export const RouterHead = component$(() => {
       {head.meta.map((m) => (
         <meta key={m.key} {...m} />
       ))}
-      {iconsLinks.map((l) => (
+      {meta.map((m) => (
```

Comment:
What's this for?

Reply:
to add `theme-color` from web manifest file (using `theme_color` option)

Reply:
nice.